noxfile.py

```diff
@@ -0,0 +1,62 @@
+import nox
+
+nox.options.reuse_existing_virtualenvs = True
+nox.options.stop_on_first_error = True
+
+deps = [
+  "setuptools-rust",
+  "fixtures",
+  "testtools>=2.5.0",
+  "networkx>=2.5",
+  "stestr",
+]
+
+extras = [
+  "mpl",
+  "graphviz",
+]
```

Comment:
To be honest, I haven't integrated this well yet. I need to check how to pass the extras for the right environments

Reply:
This was a Tox thing. I switched all the pip install to use `.[all]` to include the deps